packages/project-build/src/db/styles.ts

```diff
@@ -1,208 +1,20 @@
-import warnOnce from "warn-once";
 import { type Patch, applyPatches } from "immer";
 import { type Project, prisma } from "@webstudio-is/prisma-client";
-import type { Asset } from "@webstudio-is/asset-uploader";
-import { formatAsset } from "@webstudio-is/asset-uploader/server";
 import {
   authorizeProject,
   type AppContext,
 } from "@webstudio-is/trpc-interface/server";
 import type { Build } from "../types";
-import {
-  type StoredStyleDecl,
-  type StyleDecl,
-  getStyleDeclKey,
-  StoredStyles,
-  Styles,
-} from "../schema/styles";
-
-const parseValue = (
```

Comment:
where did all of this logic go?

Reply:
Here https://github.com/webstudio-is/webstudio-builder/pull/1405